commands/FBDisplayCommands.py

```diff
@@ -145,3 +147,28 @@ def args(self):
 
   def run(self, args, options):
     viewHelpers.setViewHidden(args[0], True)
+
+
+class FBSlowAnimationCommand(fb.FBCommand):
+  def name(self):
+    return 'slowanim'
+
+  def description(self):
+    return 'Slows down animations. Works on the iOS Simulator and a device.'
+
+  def args(self):
+    return [ fb.FBCommandArgument(arg='speed', type='float', help='Animation speed.') ]
+
+  def run(self, args, option):
+    viewHelpers.slowAnimation(args[0])
```

Comment:
Can we provide a default? If I call `slowanim`, maybe set it to 0.25? Perhaps we can dig up the speed the simulator uses, but that's hardly worth spending much time on.